config/manager/kustomization.yaml

```diff
@@ -15,5 +15,4 @@ configMapGenerator:
 
 images:
 - name: controller
-  newName: europe-docker.pkg.dev/kyma-project/prod/lifecycle-manager
```

Comment:
I don't understand this change, can you please explain how this still works?

Reply:
Good catch! This sneaked in from this Makefile command: `cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}`